diff --git a/elements/buildsystems/jobserver-pool.bst b/elements/buildsystems/jobserver-pool.bst
index 116f7db..3f0d779 100644
--- a/elements/buildsystems/jobserver-pool.bst
+++ b/elements/buildsystems/jobserver-pool.bst
@@ -7,7 +7,7 @@ sources:
   ref: v1.13.2-0-g3441b633c2fe2c494e958780ba0f4227b1327634
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 runtime-depends:
 - freedesktop-sdk.bst:components/python3.bst
diff --git a/elements/buildsystems/meson.bst b/elements/buildsystems/meson.bst
index c71b84b..a41e278 100644
--- a/elements/buildsystems/meson.bst
+++ b/elements/buildsystems/meson.bst
@@ -10,4 +10,4 @@ depends:
 - freedesktop-sdk.bst:components/ninja.bst
 - freedesktop-sdk.bst:components/pkg-config.bst
 - freedesktop-sdk.bst:components/git.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
diff --git a/elements/core-deps/slirp-helper.bst b/elements/core-deps/slirp-helper.bst
index 653de23..fa131a8 100644
--- a/elements/core-deps/slirp-helper.bst
+++ b/elements/core-deps/slirp-helper.bst
@@ -6,7 +6,7 @@ build-depends:
 - freedesktop-sdk.bst:components/rust.bst
 - freedesktop-sdk.bst:components/libffi.bst
 - freedesktop-sdk.bst:components/pkg-config.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 depends:
 - freedesktop-sdk.bst:components/libslirp.bst
diff --git a/elements/core-deps/typescript.bst b/elements/core-deps/typescript.bst
index 52745fa..74a94c3 100644
--- a/elements/core-deps/typescript.bst
+++ b/elements/core-deps/typescript.bst
@@ -6,7 +6,7 @@ sources:
   ref: 10e108c9cf7d5f2879053dff18515fb405abf2ccef63eaaf017d9c571687a1d3
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 depends:
 - core-deps/nodejs.bst
diff --git a/elements/flatpak/extensions-repo.bst b/elements/flatpak/extensions-repo.bst
index 6263dc5..80e0880 100644
--- a/elements/flatpak/extensions-repo.bst
+++ b/elements/flatpak/extensions-repo.bst
@@ -1,7 +1,7 @@
 kind: flatpak_repo
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - freedesktop-sdk.bst:components/flatpak.bst
 - filename:
   - freedesktop-sdk.bst:flatpak-images/mesa.bst
diff --git a/elements/flatpak/runtimes-repo.bst b/elements/flatpak/runtimes-repo.bst
index 342b4d0..a9546d7 100644
--- a/elements/flatpak/runtimes-repo.bst
+++ b/elements/flatpak/runtimes-repo.bst
@@ -1,7 +1,7 @@
 kind: flatpak_repo
 build-depends:
 - freedesktop-sdk.bst:components/flatpak.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 (?):
 # platform and sdk should not be built on 64bit
 - arch in ["x86_64", "aarch64", "riscv64", "ppc64le"]:
diff --git a/elements/gnomeos-deps/alsa-ucm-conf.bst b/elements/gnomeos-deps/alsa-ucm-conf.bst
index 105bdaa..464fa18 100644
--- a/elements/gnomeos-deps/alsa-ucm-conf.bst
+++ b/elements/gnomeos-deps/alsa-ucm-conf.bst
@@ -7,7 +7,7 @@ sources:
   ref: v1.2.16.1-0-gc68dcb174f432234dd224a3dc5270fa1f4856afd
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 depends:
 - freedesktop-sdk.bst:components/alsa-lib.bst
diff --git a/elements/gnomeos-deps/android-udev-rules.bst b/elements/gnomeos-deps/android-udev-rules.bst
index 07ebf49..7f866ac 100644
--- a/elements/gnomeos-deps/android-udev-rules.bst
+++ b/elements/gnomeos-deps/android-udev-rules.bst
@@ -9,7 +9,7 @@ sources:
 build-depends:
 - freedesktop-sdk.bst:components/pkg-config.bst
 - freedesktop-sdk.bst:components/systemd.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 depends:
 - freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
diff --git a/elements/gnomeos-deps/bt-set-addr.bst b/elements/gnomeos-deps/bt-set-addr.bst
index e69729e..ca9e486 100644
--- a/elements/gnomeos-deps/bt-set-addr.bst
+++ b/elements/gnomeos-deps/bt-set-addr.bst
@@ -5,7 +5,7 @@ sources:
   path: files/gnomeos/bt-set-addr
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos-deps/distrobox.bst b/elements/gnomeos-deps/distrobox.bst
index 9b7a0f8..1aabc05 100644
--- a/elements/gnomeos-deps/distrobox.bst
+++ b/elements/gnomeos-deps/distrobox.bst
@@ -9,7 +9,7 @@ sources:
   ref: 1.8.2.5-0-g40c3cd724faa434aeb0a23e28776665b92de68bd
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 runtime-depends:
 - freedesktop-sdk.bst:components/podman.bst
diff --git a/elements/gnomeos-deps/fairphone-firmware.bst b/elements/gnomeos-deps/fairphone-firmware.bst
index 42d42e6..6f54d89 100644
--- a/elements/gnomeos-deps/fairphone-firmware.bst
+++ b/elements/gnomeos-deps/fairphone-firmware.bst
@@ -7,7 +7,7 @@ sources:
   ref: a4908f548e6f88965e78b1478af1751b6a854fc9
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - gnomeos-deps/pil-squasher.bst
 
 depends:
diff --git a/elements/gnomeos-deps/grcov.bst b/elements/gnomeos-deps/grcov.bst
index b30b4f8..a29e6b9 100644
--- a/elements/gnomeos-deps/grcov.bst
+++ b/elements/gnomeos-deps/grcov.bst
@@ -812,7 +812,7 @@ build-depends:
 - freedesktop-sdk.bst:components/gcc.bst
 - freedesktop-sdk.bst:components/rust.bst
 - freedesktop-sdk.bst:components/stripper.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 depends:
 - freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
diff --git a/elements/gnomeos-deps/hidraw-udev.bst b/elements/gnomeos-deps/hidraw-udev.bst
index ef1a675..30915eb 100644
--- a/elements/gnomeos-deps/hidraw-udev.bst
+++ b/elements/gnomeos-deps/hidraw-udev.bst
@@ -1,7 +1,7 @@
 kind: manual
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos-deps/intel-ucode.bst b/elements/gnomeos-deps/intel-ucode.bst
index 73b227d..b74fb78 100644
--- a/elements/gnomeos-deps/intel-ucode.bst
+++ b/elements/gnomeos-deps/intel-ucode.bst
@@ -8,7 +8,7 @@ sources:
 
 build-depends:
 - gnomeos-deps/iucode-tool.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos-deps/journald-config.bst b/elements/gnomeos-deps/journald-config.bst
index b23055d..30249ec 100644
--- a/elements/gnomeos-deps/journald-config.bst
+++ b/elements/gnomeos-deps/journald-config.bst
@@ -1,7 +1,7 @@
 kind: manual
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 depends:
 - filename: freedesktop-sdk.bst:components/systemd.bst
diff --git a/elements/gnomeos-deps/ld-config.bst b/elements/gnomeos-deps/ld-config.bst
index d2c6d29..5adeb9b 100644
--- a/elements/gnomeos-deps/ld-config.bst
+++ b/elements/gnomeos-deps/ld-config.bst
@@ -4,7 +4,7 @@ depends:
 - gnomeos/ldconfig-always.bst
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - freedesktop-sdk.bst:components/systemd.bst
 - freedesktop-sdk.bst:components/pkg-config.bst
 
diff --git a/elements/gnomeos-deps/microcode.bst b/elements/gnomeos-deps/microcode.bst
index 54f0b0a..de84116 100644
--- a/elements/gnomeos-deps/microcode.bst
+++ b/elements/gnomeos-deps/microcode.bst
@@ -1,7 +1,7 @@
 kind: manual
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - freedesktop-sdk.bst:components/cpio.bst
 - freedesktop-sdk.bst:components/findutils.bst
 - filename:
diff --git a/elements/gnomeos-deps/modprobe-config.bst b/elements/gnomeos-deps/modprobe-config.bst
index 8fe9d52..e722ed2 100644
--- a/elements/gnomeos-deps/modprobe-config.bst
+++ b/elements/gnomeos-deps/modprobe-config.bst
@@ -5,7 +5,7 @@ sources:
   path: files/modprobe-config/soc-button-array.conf
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos-deps/nano-default-editor.bst b/elements/gnomeos-deps/nano-default-editor.bst
index 2f428e5..8b322cf 100644
--- a/elements/gnomeos-deps/nano-default-editor.bst
+++ b/elements/gnomeos-deps/nano-default-editor.bst
@@ -5,7 +5,7 @@ sources:
   path: files/gnomeos/nano-default-editor
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos-deps/noto-cjk.bst b/elements/gnomeos-deps/noto-cjk.bst
index 520b95b..5ca130e 100644
--- a/elements/gnomeos-deps/noto-cjk.bst
+++ b/elements/gnomeos-deps/noto-cjk.bst
@@ -1,7 +1,7 @@
 kind: manual
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - freedesktop-sdk.bst:components/fontconfig.bst
 - freedesktop-sdk.bst:components/pkg-config.bst
 
diff --git a/elements/gnomeos-deps/oo7-cli.bst b/elements/gnomeos-deps/oo7-cli.bst
index c587dd1..a04c44c 100644
--- a/elements/gnomeos-deps/oo7-cli.bst
+++ b/elements/gnomeos-deps/oo7-cli.bst
@@ -6,7 +6,7 @@ build-depends:
 - freedesktop-sdk.bst:components/rust.bst
 - freedesktop-sdk.bst:components/gcc.bst
 - freedesktop-sdk.bst:components/stripper.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 depends:
 - freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
diff --git a/elements/gnomeos-deps/plymouth-gnome-theme.bst b/elements/gnomeos-deps/plymouth-gnome-theme.bst
index d97f020..6de2b2a 100644
--- a/elements/gnomeos-deps/plymouth-gnome-theme.bst
+++ b/elements/gnomeos-deps/plymouth-gnome-theme.bst
@@ -8,7 +8,7 @@ runtime-depends:
 - core-deps/plymouth.bst
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos-deps/postmarketos-firmware.bst b/elements/gnomeos-deps/postmarketos-firmware.bst
index cab9347..f619abc 100644
--- a/elements/gnomeos-deps/postmarketos-firmware.bst
+++ b/elements/gnomeos-deps/postmarketos-firmware.bst
@@ -10,7 +10,7 @@ sources:
   directory: local
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos-deps/udev-hide-usr.bst b/elements/gnomeos-deps/udev-hide-usr.bst
index 74f7d3c..fa8fe81 100644
--- a/elements/gnomeos-deps/udev-hide-usr.bst
+++ b/elements/gnomeos-deps/udev-hide-usr.bst
@@ -3,7 +3,7 @@ kind: manual
 build-depends:
 - freedesktop-sdk.bst:components/systemd.bst
 - freedesktop-sdk.bst:components/pkg-config.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos-deps/vpnc-scripts.bst b/elements/gnomeos-deps/vpnc-scripts.bst
index 72b9488..660d8f1 100644
--- a/elements/gnomeos-deps/vpnc-scripts.bst
+++ b/elements/gnomeos-deps/vpnc-scripts.bst
@@ -17,4 +17,4 @@ config:
     %{install-extra}
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
diff --git a/elements/gnomeos-deps/vte-config.bst b/elements/gnomeos-deps/vte-config.bst
index 392f5a7..1d127a5 100644
--- a/elements/gnomeos-deps/vte-config.bst
+++ b/elements/gnomeos-deps/vte-config.bst
@@ -5,7 +5,7 @@ sources:
   path: files/gnomeos/vte-profile
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos-deps/words.bst b/elements/gnomeos-deps/words.bst
index 8ffe9f1..694d1d3 100644
--- a/elements/gnomeos-deps/words.bst
+++ b/elements/gnomeos-deps/words.bst
@@ -29,7 +29,7 @@ sources:
 build-depends:
 - freedesktop-sdk.bst:components/aspell.bst
 - freedesktop-sdk.bst:components/findutils.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
   
 depends:
 - freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
diff --git a/elements/gnomeos-deps/wpa-supplicant-config.bst b/elements/gnomeos-deps/wpa-supplicant-config.bst
index 3e6f426..d2c2608 100644
--- a/elements/gnomeos-deps/wpa-supplicant-config.bst
+++ b/elements/gnomeos-deps/wpa-supplicant-config.bst
@@ -7,7 +7,7 @@ sources:
 build-depends:
 - freedesktop-sdk.bst:components/pkg-config.bst
 - freedesktop-sdk.bst:components/systemd.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 runtime-depends:
 - freedesktop-sdk.bst:components/wpa-supplicant.bst
diff --git a/elements/gnomeos-deps/wsdd.bst b/elements/gnomeos-deps/wsdd.bst
index 54c79b1..d5fd34a 100644
--- a/elements/gnomeos-deps/wsdd.bst
+++ b/elements/gnomeos-deps/wsdd.bst
@@ -8,7 +8,7 @@ sources:
 
 depends:
 - freedesktop-sdk.bst:components/python3.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos/codecs-extra/filesystem.bst b/elements/gnomeos/codecs-extra/filesystem.bst
index 9c2ec17..827739e 100644
--- a/elements/gnomeos/codecs-extra/filesystem.bst
+++ b/elements/gnomeos/codecs-extra/filesystem.bst
@@ -1,7 +1,7 @@
 kind: compose
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - gnomeos/codecs-extra/deps.bst
 
 config:
diff --git a/elements/gnomeos/codecs-extra/image.bst b/elements/gnomeos/codecs-extra/image.bst
index 7c9326f..fb5b811 100644
--- a/elements/gnomeos/codecs-extra/image.bst
+++ b/elements/gnomeos/codecs-extra/image.bst
@@ -1,7 +1,7 @@
 kind: manual
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - gnomeos/save-perms.bst
 - freedesktop-sdk.bst:components/fakecap.bst
 - freedesktop-sdk.bst:components/erofs-utils.bst
diff --git a/elements/gnomeos/confext.bst b/elements/gnomeos/confext.bst
index 6c55e92..078305d 100644
--- a/elements/gnomeos/confext.bst
+++ b/elements/gnomeos/confext.bst
@@ -7,7 +7,7 @@ sources:
 build-depends:
 - freedesktop-sdk.bst:components/gcc.bst
 - freedesktop-sdk.bst:components/stripper.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 depends:
 - freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
diff --git a/elements/gnomeos/debug/image.bst b/elements/gnomeos/debug/image.bst
index ed384e9..072f39d 100644
--- a/elements/gnomeos/debug/image.bst
+++ b/elements/gnomeos/debug/image.bst
@@ -5,7 +5,7 @@ build-depends:
 - freedesktop-sdk.bst:components/fakecap.bst
 - freedesktop-sdk.bst:components/systemd.bst
 - freedesktop-sdk.bst:components/erofs-utils.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - gnomeos/os-release.bst
 - filename: gnomeos/debug/layer.bst
   config:
diff --git a/elements/gnomeos/debug/layer-tar.bst b/elements/gnomeos/debug/layer-tar.bst
index f05b310..bf7c6ce 100644
--- a/elements/gnomeos/debug/layer-tar.bst
+++ b/elements/gnomeos/debug/layer-tar.bst
@@ -1,7 +1,7 @@
 kind: script
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - freedesktop-sdk.bst:components/tar.bst
 - freedesktop-sdk.bst:components/zstd.bst
 - gnomeos/deps-all.bst
diff --git a/elements/gnomeos/debug/layer.bst b/elements/gnomeos/debug/layer.bst
index ead8ea6..5ce0755 100644
--- a/elements/gnomeos/debug/layer.bst
+++ b/elements/gnomeos/debug/layer.bst
@@ -1,7 +1,7 @@
 kind: script
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - gnomeos/save-perms.bst
 - freedesktop-sdk.bst:components/fakecap.bst
 - freedesktop-sdk.bst:components/systemd.bst
diff --git a/elements/gnomeos/devel/image.bst b/elements/gnomeos/devel/image.bst
index 3d7991f..5faec3e 100644
--- a/elements/gnomeos/devel/image.bst
+++ b/elements/gnomeos/devel/image.bst
@@ -6,7 +6,7 @@ build-depends:
 - freedesktop-sdk.bst:components/erofs-utils.bst
 - freedesktop-sdk.bst:components/jq.bst
 - freedesktop-sdk.bst:components/systemd.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - filename: gnomeos/devel/layer.bst
   config:
     location: '/sysroot'
diff --git a/elements/gnomeos/efi-keys.bst b/elements/gnomeos/efi-keys.bst
index 5837853..b5bf70b 100644
--- a/elements/gnomeos/efi-keys.bst
+++ b/elements/gnomeos/efi-keys.bst
@@ -33,7 +33,7 @@ sources:
 
 build-depends:
 - freedesktop-sdk.bst:components/sbsigntools-maybe.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   gnomeos-keys-uuid: 8dd6b2b4-3dc4-4ee2-a226-635467997f0e
diff --git a/elements/gnomeos/fwupd-efi-signed.bst b/elements/gnomeos/fwupd-efi-signed.bst
index 35eaa41..dccefd5 100644
--- a/elements/gnomeos/fwupd-efi-signed.bst
+++ b/elements/gnomeos/fwupd-efi-signed.bst
@@ -2,7 +2,7 @@ kind: manual
 
 build-depends:
 - freedesktop-sdk.bst:components/sbsigntools-maybe.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - core-deps/fwupd-efi.bst
 
 depends:
diff --git a/elements/gnomeos/generate-initramfs.bst b/elements/gnomeos/generate-initramfs.bst
index f31c1c4..3a6fc04 100644
--- a/elements/gnomeos/generate-initramfs.bst
+++ b/elements/gnomeos/generate-initramfs.bst
@@ -5,7 +5,7 @@ sources:
   path: files/gnomeos/generate-initramfs
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 runtime-depends:
 - gnomeos-deps/python3-zstd.bst
diff --git a/elements/gnomeos/initramfs/deps.bst b/elements/gnomeos/initramfs/deps.bst
index f76a5b4..8f8b70e 100644
--- a/elements/gnomeos/initramfs/deps.bst
+++ b/elements/gnomeos/initramfs/deps.bst
@@ -1,7 +1,7 @@
 kind: stack
 
 depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - freedesktop-sdk.bst:components/btrfs-progs.bst
 - freedesktop-sdk.bst:components/cpio.bst
 - freedesktop-sdk.bst:components/dbus.bst
diff --git a/elements/gnomeos/initramfs/image.bst b/elements/gnomeos/initramfs/image.bst
index d8f8351..8ac1868 100644
--- a/elements/gnomeos/initramfs/image.bst
+++ b/elements/gnomeos/initramfs/image.bst
@@ -1,7 +1,7 @@
 kind: manual
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - freedesktop-sdk.bst:components/cpio.bst
 - freedesktop-sdk.bst:components/findutils.bst
 - freedesktop-sdk.bst:components/zstd.bst
diff --git a/elements/gnomeos/initramfs/signed-modules.bst b/elements/gnomeos/initramfs/signed-modules.bst
index 783ab6d..f52620b 100644
--- a/elements/gnomeos/initramfs/signed-modules.bst
+++ b/elements/gnomeos/initramfs/signed-modules.bst
@@ -6,7 +6,7 @@ build-depends:
 - freedesktop-sdk.bst:components/findutils.bst
 - freedesktop-sdk.bst:components/sign-file.bst
 - freedesktop-sdk.bst:components/zstd.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 config:
   install-commands:
diff --git a/elements/gnomeos/initramfs/signed-nvidia-modules.bst b/elements/gnomeos/initramfs/signed-nvidia-modules.bst
index ff228de..c489065 100644
--- a/elements/gnomeos/initramfs/signed-nvidia-modules.bst
+++ b/elements/gnomeos/initramfs/signed-nvidia-modules.bst
@@ -6,7 +6,7 @@ build-depends:
 - freedesktop-sdk.bst:components/findutils.bst
 - freedesktop-sdk.bst:components/sign-file.bst
 - freedesktop-sdk.bst:components/zstd.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos/live-image.bst b/elements/gnomeos/live-image.bst
index 460c842..851343b 100644
--- a/elements/gnomeos/live-image.bst
+++ b/elements/gnomeos/live-image.bst
@@ -1,7 +1,7 @@
 kind: script
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - freedesktop-sdk.bst:components/jq.bst
 - freedesktop-sdk.bst:components/systemd.bst
 - freedesktop-sdk.bst:components/dosfstools.bst
diff --git a/elements/gnomeos/make-layer.bst b/elements/gnomeos/make-layer.bst
index b3e33a2..29b5f05 100644
--- a/elements/gnomeos/make-layer.bst
+++ b/elements/gnomeos/make-layer.bst
@@ -3,7 +3,7 @@ kind: manual
 depends:
 - freedesktop-sdk.bst:components/python3.bst
 - freedesktop-sdk.bst:components/erofs-utils.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos/nvidia-modules/image.bst b/elements/gnomeos/nvidia-modules/image.bst
index ea03d4d..3ff5a93 100644
--- a/elements/gnomeos/nvidia-modules/image.bst
+++ b/elements/gnomeos/nvidia-modules/image.bst
@@ -5,7 +5,7 @@ build-depends:
 - freedesktop-sdk.bst:components/fakecap.bst
 - freedesktop-sdk.bst:components/systemd.bst
 - freedesktop-sdk.bst:components/erofs-utils.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - gnomeos/os-release.bst
 - filename: gnomeos/nvidia-modules/layer.bst
   config:
diff --git a/elements/gnomeos/nvidia-modules/layer.bst b/elements/gnomeos/nvidia-modules/layer.bst
index 022d79b..666efbe 100644
--- a/elements/gnomeos/nvidia-modules/layer.bst
+++ b/elements/gnomeos/nvidia-modules/layer.bst
@@ -1,7 +1,7 @@
 kind: script
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - gnomeos/save-perms.bst
 - freedesktop-sdk.bst:components/fakecap.bst
 - freedesktop-sdk.bst:components/systemd.bst
diff --git a/elements/gnomeos/nvidia-runtime/image.bst b/elements/gnomeos/nvidia-runtime/image.bst
index f6ba968..18a0b43 100644
--- a/elements/gnomeos/nvidia-runtime/image.bst
+++ b/elements/gnomeos/nvidia-runtime/image.bst
@@ -6,7 +6,7 @@ build-depends:
 - freedesktop-sdk.bst:components/erofs-utils.bst
 - freedesktop-sdk.bst:components/jq.bst
 - freedesktop-sdk.bst:components/systemd.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - filename: gnomeos/nvidia-runtime/layer.bst
   config:
     location: '/sysroot'
diff --git a/elements/gnomeos/public-keys.bst b/elements/gnomeos/public-keys.bst
index 0e0578e..73c4a03 100644
--- a/elements/gnomeos/public-keys.bst
+++ b/elements/gnomeos/public-keys.bst
@@ -7,7 +7,7 @@ sources:
   path: files/boot-keys/fstab-tpm2-pcr-public.pem
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos/replace-signed-systemd-boot.bst b/elements/gnomeos/replace-signed-systemd-boot.bst
index 35e188f..1e169f5 100644
--- a/elements/gnomeos/replace-signed-systemd-boot.bst
+++ b/elements/gnomeos/replace-signed-systemd-boot.bst
@@ -2,7 +2,7 @@ kind: manual
 
 build-depends:
 - freedesktop-sdk.bst:components/sbsigntools-maybe.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 depends:
 - filename: freedesktop-sdk.bst:components/systemd.bst
diff --git a/elements/gnomeos/signed-boot-common.bst b/elements/gnomeos/signed-boot-common.bst
index 75294ee..1042d55 100644
--- a/elements/gnomeos/signed-boot-common.bst
+++ b/elements/gnomeos/signed-boot-common.bst
@@ -7,7 +7,7 @@ build-depends:
   - gnomeos/replace-signed-systemd-boot.bst
   strict: true
 - freedesktop-sdk.bst:components/sbsigntools-maybe.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos/signed-boot.bst b/elements/gnomeos/signed-boot.bst
index 5567da9..98b225d 100644
--- a/elements/gnomeos/signed-boot.bst
+++ b/elements/gnomeos/signed-boot.bst
@@ -16,7 +16,7 @@ build-depends:
 - freedesktop-sdk.bst:components/sbsigntools-maybe.bst
 - freedesktop-sdk.bst:components/tpm2-tss.bst
 - freedesktop-sdk.bst:components/util-linux.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 (?):
 - arch == "x86_64":
diff --git a/elements/gnomeos/snapd/image.bst b/elements/gnomeos/snapd/image.bst
index e61d26d..ed301ed 100644
--- a/elements/gnomeos/snapd/image.bst
+++ b/elements/gnomeos/snapd/image.bst
@@ -6,7 +6,7 @@ build-depends:
 - freedesktop-sdk.bst:components/erofs-utils.bst
 - freedesktop-sdk.bst:components/jq.bst
 - freedesktop-sdk.bst:components/systemd.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - filename: gnomeos/snapd/layer.bst
   config:
     location: '/sysroot'
diff --git a/elements/gnomeos/snapd/snapd-services.bst b/elements/gnomeos/snapd/snapd-services.bst
index c7d0978..b909d78 100644
--- a/elements/gnomeos/snapd/snapd-services.bst
+++ b/elements/gnomeos/snapd/snapd-services.bst
@@ -3,7 +3,7 @@ kind: manual
 build-depends:
 - freedesktop-sdk.bst:components/pkg-config.bst
 - freedesktop-sdk.bst:components/systemd.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos/systemd-pcrlock-workaround.bst b/elements/gnomeos/systemd-pcrlock-workaround.bst
index effcfce..aece0d6 100644
--- a/elements/gnomeos/systemd-pcrlock-workaround.bst
+++ b/elements/gnomeos/systemd-pcrlock-workaround.bst
@@ -5,7 +5,7 @@ sources:
   path: files/gnomeos/systemd-pcrlock-workaround.conf
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/gnomeos/update-images-user-only.bst b/elements/gnomeos/update-images-user-only.bst
index 3a07022..744d767 100644
--- a/elements/gnomeos/update-images-user-only.bst
+++ b/elements/gnomeos/update-images-user-only.bst
@@ -8,7 +8,7 @@ build-depends:
 - filename: gnomeos/signed-boot.bst
   config:
     location: /signed-boot
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 config:
   commands:
diff --git a/elements/gnomeos/update-images.bst b/elements/gnomeos/update-images.bst
index 0eb4128..f3d9fba 100644
--- a/elements/gnomeos/update-images.bst
+++ b/elements/gnomeos/update-images.bst
@@ -23,7 +23,7 @@ build-depends:
 - filename: gnomeos/codecs-extra/image.bst
   config:
     location: "%{install-root}"
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 (?):
 - arch in ["x86_64"]:
diff --git a/elements/oci/initramfs/deps.bst b/elements/oci/initramfs/deps.bst
index 19c0335..9525abe 100644
--- a/elements/oci/initramfs/deps.bst
+++ b/elements/oci/initramfs/deps.bst
@@ -1,7 +1,7 @@
 kind: stack
 
 depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - freedesktop-sdk.bst:components/btrfs-progs.bst
 - freedesktop-sdk.bst:components/cpio.bst
 - freedesktop-sdk.bst:components/dbus.bst
diff --git a/elements/oci/initramfs/image.bst b/elements/oci/initramfs/image.bst
index 964406f..9928d13 100644
--- a/elements/oci/initramfs/image.bst
+++ b/elements/oci/initramfs/image.bst
@@ -1,7 +1,7 @@
 kind: manual
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - freedesktop-sdk.bst:components/cpio.bst
 - freedesktop-sdk.bst:components/findutils.bst
 - freedesktop-sdk.bst:components/zstd.bst
diff --git a/elements/oci/integration/bootc-config.bst b/elements/oci/integration/bootc-config.bst
index f769424..bc869a4 100644
--- a/elements/oci/integration/bootc-config.bst
+++ b/elements/oci/integration/bootc-config.bst
@@ -5,7 +5,7 @@ sources:
   path: files/oci
 
 depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 variables:
   strip-binaries: ''
diff --git a/elements/oci/integration/cleanup-debug.bst b/elements/oci/integration/cleanup-debug.bst
index ae93cc1..2addf4b 100644
--- a/elements/oci/integration/cleanup-debug.bst
+++ b/elements/oci/integration/cleanup-debug.bst
@@ -1,7 +1,7 @@
 kind: stack
 
 depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 public:
   bst:
diff --git a/elements/oci/integration/extrafs.bst b/elements/oci/integration/extrafs.bst
index fffb6d1..5059606 100644
--- a/elements/oci/integration/extrafs.bst
+++ b/elements/oci/integration/extrafs.bst
@@ -6,7 +6,7 @@
 kind: script
 
 build-depends:
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 
 config:
   commands:
diff --git a/elements/oci/platform/stack.bst b/elements/oci/platform/stack.bst
index e050b82..c9b307f 100644
--- a/elements/oci/platform/stack.bst
+++ b/elements/oci/platform/stack.bst
@@ -4,6 +4,6 @@ depends:
 - sdk-platform.bst
 - sdk/os-release.bst
 - oci/integration/extrafs.bst
-- freedesktop-sdk.bst:public-stacks/runtime-gnu.bst
+- freedesktop-sdk.bst:public-stacks/runtime-minimal.bst
 - freedesktop-sdk.bst:vm/mesa-default.bst
 
