--- a/src/buildstream_plugins_community/sources/_git_utils.py
+++ b/src/buildstream_plugins_community/sources/_git_utils.py
@@ -100,9 +100,9 @@ def collect_objects(mirror, commit, base=None):
         excluded = {commit for commit in base.parents if commit in mirror}
         commits = {
             entry.commit
             for entry in mirror.get_walker(commit.id, exclude=excluded)
         }
 
     # All the tree objects. `git log .` and `git describe --dirty` need them
-    trees = set(commit.tree for commit in commits)
+    trees = set(commit.tree for commit in commits if hasattr(commit, 'tree'))
     objects = set()
 
     while trees:
